tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -25,6 +25,12 @@ drop_packets/test_drop_counters.py::test_ip_is_zero_addr:
     conditions:
       - "asic_type=='cisco-8000'"
 
+dut_console/test_escape_character.py::test_console_escape:
```

Comment:
Mark test case to be skipped on vs is OK, but not necessary. Normally, the test case will be run on virtual testbed when we add it to `kvm_test.sh`. 
I suggest adding some check for vs testbed in testing code if the test case is not designed for virtual testbed.